cmd/immudb/command/init.go

```diff
@@ -58,6 +58,7 @@ func (cl *Commandline) setupFlags(cmd *cobra.Command, options *server.Options) {
 	cmd.Flags().String("clientcas", "", "clients certificates list. Aka certificate authority")
 	cmd.Flags().Bool("devmode", options.DevMode, "enable dev mode: accept remote connections without auth")
 	cmd.Flags().String("admin-password", options.AdminPassword, "admin password (default is 'immudb') as plain-text or base64 encoded (must be prefixed with 'enc:' if it is encoded)")
+	cmd.Flags().Bool("force-admin-password", false, "if true, reset the admin password is to the one passed through `admin-password` option if it was changed")
```

Comment:
what about always overwriting it if `admin-password` is specified? otherwise it will just use current one (or default if the user needs to be created). So we don't need to add an extra flag

Reply:
That would be a pretty nasty backwards incompatibility. There could be some services out there where the admin password is set on the first run but the option is still there in subsequent runs (e.g. to avoid having a dedicated first-run operation).

A separate option means we can maintain backwards compatibility.